botorch/models/approximate_gp.py

```diff
@@ -211,23 +211,7 @@ def __init__(
                 # as a heuristic
                 inducing_points = int(0.25 * train_X.shape[-2])
 
-            with torch.no_grad():
-                train_train_kernel = covar_module(train_X)
-                if train_train_kernel.ndimension() > 2:
-                    train_train_kernel = train_train_kernel.evaluate_kernel()[0]
-                    need_to_repeat_over_batch = True
-                else:
-                    need_to_repeat_over_batch = False
-
-                inducing_points = _pivoted_cholesky_init(
-                    train_X, train_train_kernel, max_length=inducing_points
-                )
-
-                if need_to_repeat_over_batch:
-                    inducing_points = inducing_points.unsqueeze(0)
-                    inducing_points = inducing_points.repeat(
-                        *batch_shape, *([1] * (inducing_points.ndimension() - 1))
-                    )
+            inducing_points = _select_inducing_points(train_X, covar_module, inducing_points, batch_shape)
```

Comment:
Did you mean to call `init_inducing_points` here?

Reply:
at the moment, `init_inducing_points` is a method for `SingleTaskVariationalGP` not `_SingleTaskVariationalGP`.  Passing the inducing points to the variational strategy constructor works fine here.